spec/features/admin/procedure_creation_spec.rb

```diff
@@ -97,12 +102,12 @@
       page.first('.gestionnaire-affectation').click
 
       page.find_by_id('onglet-infos').click
-      expect(page).to have_selector('#publish-procedure', visible: true)
-      page.find_by_id('publish-procedure').click
+      expect(page).to have_selector('#test-procedure', visible: true)
+      page.find_by_id('test-procedure').click
 
       expect(page.find_by_id('procedure_path')['value']).to eq('libelle-de-la-procedure')
       page.find_by_id('publish').click
-      expect(page).to have_selector('.procedure-lien')
+      expect(page).to have_current_path(admin_procedures_testing_path)
```

Comment:
Ça manque sans doute de deux tests de bout en bout :

- `scenario "I can publish a procedure for testing"`
- `scenario "I can publish a tested procedure for production"`

Reply:
ce n'est pas la manière don't les testes sont organisé aujourd'hui :( Ça devrait, mais ce n’est pas trivial. J'ai ajouté les étapes de publication en production suit à la publication en test dans le test existant.